src/main/java/io/debezium/connector/yugabytedb/connection/YugabyteDBConnection.java

```diff
@@ -61,15 +61,17 @@ public class YugabyteDBConnection extends JdbcConnection {
 
     private static Logger LOGGER = LoggerFactory.getLogger(YugabyteDBConnection.class);
 
-    private static final String URL_PATTERN = "jdbc:postgresql://${" + JdbcConfiguration.HOSTNAME + "}:${"
+    private static final String MULTI_HOST_URL_PATTERN = "jdbc:postgresql://${" + JdbcConfiguration.HOSTNAME + "}/${" + JdbcConfiguration.DATABASE + "}";
```

Comment:
I think we should be using our smart driver rather than the one from Postgres.

Reply:
IIRC we had decided to drop smart driver because it needed a lot of changes to be made. @suranjan  should we use smart driver or can we continue with postgres one?